ui/src/pages/sites/index.tsx

```diff
@@ -26,21 +26,30 @@ export default function SitesPage() {
             <h2 className="margin-top-0 text-center margin-auto">How It Works</h2>
             <ol className="grid" style={{ listStyle: 'none' }}>
               <li className="card card--lightBlue">
+                <div className="card card--icon card--elevation-2">
+                  <Icon name="brush-monitor" />
+                </div>
                 <h5>Step 1</h5>
                 <p>
                   Customize your site’s design with the tool below. Choose one of our hand-crafted website templates,
                   then add your signature styles, colors, and logos to make it your own.
                 </p>
               </li>
               <li className="card card--lightBlue">
-                <h5>Step 1</h5>
+                <div className="card card--icon card--elevation-2">
+                  <Icon name="bubble-chat" />
+                </div>
+                <h5>Step 2</h5>
```

Comment:
I havent seen the UI but I am guessing from looking at the code that your h5s are going to need a className="margin-top-small" because there is most likely too much space between the icon and title. (I could be wrong) Just make sure the mockups match the UI though